Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -86,6 +86,11 @@ WI.FontStyles = class FontStyles
     {
         switch (tag) {
         case "wdth":
+            return parseFloat(value);
+        case "ital":
+            if (value === "normal")
+                return 0;
+
             return parseFloat(value);
```

Comment:
Should this case just use the same logic as `slnt`? Otherwise it seems like we would still fail to parse the oblique keyword/degree values?

Reply:
Also do we need to similarly support `ital` in the other direction in `fontPropertyValueToAxisValue`?

Reply:
Yes, you are right. The cases for `ital` and `slnt` have significant overlap. I merged the two and adapted accordingly. 

Also added mapping `ital` value to `font-style: italic | normal` in `fontPropertyValueToAxisValue` under the assumptions [per spec](https://w3c.github.io/csswg-drafts/css-fonts/#valdef-font-style-oblique-angle--90deg-90deg) that an `ital` axis is supposed to act as an on/off switch and its value is expected to be in the range 0–1 (0 = off, 1 = on). 

Checked with @litherum and the guidance is to not do anything special for non-conforming fonts with ranges beyond 0–1. Chose to map anything `ital` value >=1  to `font-style: italic`

Added tests for these mapping functions as they become more complex.